src/extension.ts

```diff
@@ -122,7 +122,8 @@ async function serverOpenConfig(tomcat: TomcatController, tomcatItem ?: TomcatSe
 
 async function createServer(tomcat: TomcatController): Promise<string> {
     const ui: VSCodeUI = new VSCodeUI();
-    const tomcatPath: string = await selectFolder(ui, Utility.localize('tomcatExt.selectinstllpath', 'Select Tomcat Installation Path'));
+    const tomcatPath: string = await ui.showFileFolderDialog(false, true, Utility.localize('tomcatExt.selectserver', 'Select Tomcat Server'));
```

Comment:
Why to change 'Select Tomcat Installation Path' to 'Select Tomcat Server'

Reply:
Previous behavior uses Select Tomcat Installation Path as hint but user have to do one more click the "Browse...":
![click1](https://user-images.githubusercontent.com/5179063/34600231-641aab6e-f231-11e7-83f8-e359bdcf891c.png)

Update the behavior to open File explorer directly with hint to select folder to save user one button click effort:
![click2](https://user-images.githubusercontent.com/5179063/34600263-8e0bd970-f231-11e7-87eb-951a0a43e5ad.png)

Reply:
I don't think the phrase "select tomcat server" is proper for the folder selector, keep the original label:  "Select Tomcat Installation Path" or "Select Tomcat Directory" for the folder selector for the hint of a tomcat location/folder.

Reply:
I will take the "Select Tomcat Directory".